Containerfile.common

```diff
@@ -61,6 +61,17 @@ RUN --mount=type=cache,dst=/var/cache/dnf \
     done && \
     find /var/cache/rpms
 
+# Remove kernel version from kmod package names
+# FIXME: This will only work for 6.* kernels unfortunately
```

Comment:
I have no concerns about "6.* kernels only" for ublue specific builds... we aren't going back to 5 series.

Reply:
Was a reference to the fact it will break when kernels advance to 7 heh